dev-tools/release-notes/main/main.go

```diff
@@ -0,0 +1,266 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Could we reuse one of:
- `.ci/`
- `build/`
- `operators/hack/`
instead of creating a new `dev-tools` main dir?

Reply:
Yes absolutely. wasn't sure about the the best place to put this.